src/Components/backtestingControls/AnalysisLauncher.tsx

```diff
@@ -57,22 +57,9 @@ const AnalysisLauncher: React.FC<AnalysisLauncherProps> = ({
         disabled={isAnalyzing}
         className="bg-blue-500 text-white p-2 rounded"
       >
-        {isAnalyzing ? "Analysis in progress..." : "Start analysis"}
+        {isAnalyzing || isLoading ? "Analysis in progress..." : "Start analysis"}
```

Comment:
seems like a mismatch with the var names / usage

Reply:
modification => {isAnalyzing ? "Analysis in progress..." : "Start analysis"}